### PR TITLE
Add Upgrading Page

### PR DESCRIPTION
**Pull request by vmitchell85**
This PR creates a new page to contain details on upgrading servers. This was based on some tweets by James Brooks. Some others have "beaten me to the punch" but since this is a slightly different take I figured I'd submit it anyway.

Thanks for considering.

### CODE REVIEW

File: 1.0/servers/upgrading.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Upgrading
+
+[[toc]]
+
+## Node.js
+
+The latest version of Node.js is installed by Forge when it is provisioning a new server. However, as your server ages, you may wish to upgrade to the latest version of Node.js:
+
+```bash
+curl -fsSL https://deb.nodesource.com/setup_current.x | sudo -E bash - sudo apt-get install -y nodejs
```
Comment (benholmen) [inline 1.0/servers/upgrading.md:10]: This needs to be split into two commands:
```suggestion
curl -fsSL https://deb.nodesource.com/setup_current.x | sudo -E bash -
sudo apt-get install -y nodejs
```

File: 1.0/servers/upgrading.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+```
+
```
Comment (benholmen) [inline 1.0/servers/upgrading.md:12]: Consider adding npm documentation from #74!

https://github.com/laravel/forge-docs/pull/74/files#diff-f0903ed5e76cc386766b600a7302aa6d5732c17ba22cb48f7f09632fe9f5547aR34

**Comment by benholmen** [comment]
@vmitchell85 I like the idea of a separate upgrading page!

**Comment by taylorotwell** [comment]
Will just leave it in the cookbook for now.

**Pull request closed without merging**